packages/ui/src/components/logo/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './logo'
+export * from './logo-name-map'
```

Comment:
Do we need `LogoNameMap` in the publicly exposed API of the library?
```suggestion
```

I'd consider removing this `index` file and just changing the import in `src/components/index.ts` to instead be `export * from './logo/logo'`, though it's fine to keep the `index.ts` file as well if you prefer the optics.

Reply:
Done as suggested.